src/components/ProgressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,59 @@
+import styled from "@emotion/styled";
+import React, { FunctionComponent } from "react";
+import { vars } from "../../styles";
+
+interface ProgressBarProps {
+  step: number;
+  className?: string;
+}
+
+export const ProgressBar: FunctionComponent<ProgressBarProps> = styled(({ className, step }) => {
+  // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+  const description = (step: number) => {
+    switch (step) {
+      case 1:
+        return "Upload Config";
+
+      case 2:
+        return "Choose Type";
+
+      case 3:
+        return "Fill Form";
+
+      case 4:
+        return "Issue complete";
+
+      default:
+        return "";
+    }
+  };
+
+  return (
+    <div className={className}>
+      <div className="steps">{`Step ${step}/4: ${description(step)}`}</div>
+      <div className="unfilled">
+        <div className="filled" />
+      </div>
+    </div>
+  );
+})`
+  .steps {
+    color: ${vars.greyDark};
+    font-weight: bold;
+    font-size: 16px;
+  }
+
+  .unfilled {
+    width: 388px;
+    height: 3px;
```

Comment:
width: 100%?

Reply:
100% will be the whole wrapper, will look like the button length, but the design not 100% so i follow the design, should i change it to 100%?

Reply:
ah i see, my issue is by defining a fixed 388px will not be responsive. in this case u can do maybe say `width: 100%; max-width: 388px;` wdyt?

Reply:
ahh ok understood!